src/Passwordless/Models/Credential.cs

```diff
@@ -1,7 +1,14 @@
 ﻿namespace Passwordless;
 
+/// <summary>
+/// Store credential used to authenticate users. Follows WebAuthN specification
+/// <see href="https://www.w3.org/TR/webauthn-2/#public-key-credential"/>
```

Comment:
I wouldn't say we actually follow the specification, we add stuff like nickname, device, country, userid etc.

I would say "A passkey credential stored by Passwordless"